config/freeradius2/privacyidea-perl

```diff
@@ -1,3 +1,3 @@
-perl {
+perl privacyidea-perl {
     module = /usr/share/privacyidea/freeradius/privacyidea_radius.pm
 }
```

Comment:
I would like to keep the file name indicating that this is a module.
Please rename the file to contain mods or modules.

Reply:
fixed in 82d037a

Reply:
fixed in 82d037a